<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-KDQEA51" failures="1" tests="1" name="UserManagement" time="22.218" errors="0" timestamp="13 Jan 2022 06:31:04 GMT">
  <testcase classname="Automation.SeleniumFramework.tests" name="Login" time="5.955">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Automation.SeleniumFramework.tests.Login(tests.java:37)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="Automation.SeleniumFramework.tests" name="windowOperations" time="0.0">
    <ignored/>
  </testcase> <!-- windowOperations -->
  <testcase classname="Automation.SeleniumFramework.tests" name="selectBtn" time="0.0">
    <ignored/>
  </testcase> <!-- selectBtn -->
</testsuite> <!-- UserManagement -->
